### ISSUE TITLE
Dates not translated

### ISSUE THREAD

**Issue by oualidfouad** [mdjnelson/moodle-mod_customcert#551]
Hi,
The dates component doesn't come up in French while the certificate is (or should be).
How do you set it up?
![frdates](https://user-images.githubusercontent.com/19862730/234623380-550d2cf8-bedf-42a1-9c1d-ace530c62ad3.jpg)

**Comment by mdjnelson** [comment]
Are you able to provide replication steps so I can reproduce this? Thanks.

**Comment by oualidfouad** [comment]
Yes, this certificate was made in a French information page with French courses, but we can't find a way to tell the plugin that the autogenerated dates should be in French as well, and it doesn't come as French automatically.
